### ISSUE TITLE
Negating the minimum value of a twos complement number is invalid

### ISSUE THREAD

**Issue by DennisGaida** [nickbabcock/OhmGraphite#353]
I got an exception when starting up OhmGraphite on a laptop, debug logging doesn't turn up more than the error:

```
2023-02-16 12:31:29.0247|ERROR|OhmGraphite.PrometheusCollection|Exception: prometheus update metrics!System.OverflowException: Negating the minimum value of a twos complement number is invalid.
   at LibreHardwareMonitor.Hardware.Battery.Battery.Update()
   at OhmGraphite.UpdateVisitor.VisitHardware(IHardware hardware)
   at LibreHardwareMonitor.Hardware.Hardware.Accept(IVisitor visitor)
   at LibreHardwareMonitor.Hardware.Computer.Traverse(IVisitor visitor)
   at OhmGraphite.UpdateVisitor.VisitComputer(IComputer computer)
   at LibreHardwareMonitor.Hardware.Computer.Accept(IVisitor visitor)
   at OhmGraphite.SensorCollector.ReadAllSensors()
   at OhmGraphite.PrometheusCollection.PollSensors()
   at OhmGraphite.LoggerUtils.LogAction(Logger logger, String msg, Action action)
```

I can circumvent this error by the following `app` setting:
`<add key="/battery/enabled" value="false" />`

I didn't look into the implementation of OhmGraphite or LibreHardwareMonitor, but I guess my battery returns some unexpected value. I am not sure whether this is a LibreHardwareMonitor issue or OhmGraphite.

**Comment by nickbabcock** [comment]
Thanks for the report, I have a hunch for solution, and I've reported it upstream.

**Comment by nickbabcock** [comment]
This should be fixed in v0.29.0, let me know if it is not
